connect-button/src/test/java/com/ifttt/connect/ui/ButtonApiHelperTest.java

```diff
@@ -25,7 +26,7 @@
     @Test
     public void testRequiredFields() {
         Uri uri = ButtonApiHelper.getEmbedUri(connection, Initial, redirectUri, Collections.emptyList(), "abc@efg.com",
-                null, "", "auth_code", null, false
+                null, "", "auth_code", null, false, "en"
```

Comment:
I don't think we have an `en` language_tag value, the default is empty, so might as well changing all these to empty string. Not a blocker.